src/DeploymentProvider/AzureSpringAppsDeploymentProvider.ts

```diff
@@ -39,14 +39,11 @@ export class AzureSpringAppsDeploymentProvider {
             }
             this.params.resourceGroupName = match[0];
             console.log('service resource group name: ' + this.params.resourceGroupName);
-        } else { //Should never ever ever happen
+        } else { //Should never happen
             throw new Error('DuplicateAzureSpringAppsName: ' + this.params.serviceName);
         }
         const serviceResponse = await this.client.services.get(this.params.resourceGroupName, this.params.serviceName);
-        core.debug("service response: " + serviceResponse._response.bodyAsText);
-        if (serviceResponse._response.status != 200) {
-            throw Error('GetServiceError: ' + this.params.serviceName);
```

Comment:
Just curious, how do we handle non-200 status code now?

Reply:
As the lastest SDK will throw Error when the response cannot be deserialized to the target type, it will throw Error when we get non-200 status response. I have tried some Error and it seems to be user-friendly. So I think we don't have to wrap the Error once more. Example as follows.
![image](https://user-images.githubusercontent.com/86712051/188813663-af4f91af-3c6f-449d-a141-dc4885a60025.png)